core/src/com/po/fuck/model/constants/GameConstants.java

```diff
@@ -0,0 +1,15 @@
+package com.po.fuck.model.constants;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import static com.po.fuck.model.constants.ConstantsLoader.loadProperties;
+import static com.po.fuck.model.constants.ConstantsLoader.loadInt;
+
+public final class GameConstants extends BaseConstants {
```

Comment:
`GameConstants` should be renamed IMO, because there are screen constants/etc

Reply:
`Game` -> `GameView`?

Reply:
Also not very great.